zubbi/models.py

```diff
@@ -352,10 +368,12 @@ def init_elasticsearch_con(
 
     # Somehow the SSL context is not enough, we must also pass the use_ssl=True
     use_ssl = ssl_context is not None
+
+    if connection_params is None:
+        connection_params = {}
```

Comment:
Can be removed as we init the connection params in line 350

Reply:
Thanks, done.